src/test/java/com/atlassian/bitbucket/jenkins/internal/status/BitbucketBuildStatusFactoryTest.java

```diff
@@ -19,98 +19,150 @@
 
 import static org.hamcrest.CoreMatchers.*;
 import static org.junit.Assert.assertThat;
-import static org.mockito.Mockito.mock;
-import static org.mockito.Mockito.when;
+import static org.mockito.Mockito.*;
 
 @RunWith(MockitoJUnitRunner.class)
 public class BitbucketBuildStatusFactoryTest {
 
-    private static final String BUILD_DISPLAY_NAME = "#15";
     private static final String BUILD_DURATION = "400 secs";
+    private static final String BUILD_ID = "15";
+    private static final String BUILD_DISPLAY_NAME = "#" + BUILD_ID;
     private static final String BUILD_URL = "http://www.example.com:8000";
-    private static final String PROJECT_NAME = "Project Name";
+    private static final String FREESTYLE_PROJECT_NAME = "Freestyle Project";
+    private static final String WORKFLOW_JOB_NAME = "branch-name";
+    private static final String WORKFLOW_PROJECT_NAME = "MultiBranch Project";
 
     @Rule
     public final JenkinsRule jenkinsRule = new JenkinsRule();
+    private BitbucketBuildStatusFactory buildStatusFactory;
     @Mock
-    private AbstractBuild build;
+    private BitbucketCICapabilities ciCapabilities;
+    @Mock
+    private Run freeStyleRun;
+    private Job freeStyleProject;
     @Mock
     private Jenkins parent;
     @Mock
-    private Project project;
+    private Run workflowRun;
+    private Job workflowJob;
     @Mock
-    private BitbucketCICapabilities ciCapabilities;
+    private MultiBranchProject<?, ?> workflowProject;
```

Comment:
You've already done what I mentioned above here, so to be consistent, let's use the `?` generics for all generic types.